README.md

```diff
@@ -8,41 +8,17 @@ makes it easy to cancel tasks, retry failed tasks, and re-run previously complet
-
-### New Features
-
-* Notifications for important task- and workflow-related events. Every workflow has a reference to a
-  notification center — by default, the default notification center — that it and its tasks use to 
-  post notifications. Tasks post notifications when they start, finish, fail, cancel, retry, and
-  reset. Notifications are also posted when workflows start, retry, and reset, as well as when all
-  their tasks finish, or a single task fails or is cancelled.
-* Delegate messages for workflow and task cancellation. 
-* Workflows can now be reset by sending them the `‑reset` message.
-* The new `TSKTask` subclass `TSKSubworkflowTask` lets you run an entire workflow as a single task.
-* Convenience methods for accessing a task’s prerequisites results.
-* Unit tests! We now have unit tests for all major functionality in the framework.
-
-### Changes
-
-* `TSKGraph` has been renamed to `TSKWorkflow` to more clearly express its purpose. All references
-  to “graphs” have been replaced with references to “workflows.” To adapt to this change, we
-  recommend you find & replace `graph` with `workflow` and `Graph` with `Workflow` for all your
-  task-related code. This is what we did to make the change.
-* Tasks no longer automatically start themselves after resetting. If you need to start a task after
-  resetting it, send it the `‑start` message immediately after sending it `‑reset`.
-* Retrying tasks that are in the pending or ready states no longer has any effect on the task itself,
-  though the dependent tasks still receive the `‑retry` message. If you wish to run a task that is
-  in the ready state, send it the `‑start` message. Tasks in the pending state cannot be run, but
-  will automatically be started when their prerequisites finish successfully.
-* Workflows that contain no tasks will finish immediately upon receiving `‑start`.
-* As a minor optimization, sending `‑start` to a task only enqueues an operation on its operation 
-  queue if the task is in the ready state. The operation itself still ensures that the task is in
-  the ready state before invoking the task’s `‑main` method, but this should lead to better
-  performance when a task’s operation queue has limited concurrency.
+## What’s New in 1.1
+
+Task 1.1 adds keyed prerequisites so that a task can retrieve its prerequisite’s results using 
+unique identifiers. This simplifies cases where a task has multiple prerequisites that it needs
+to treat specially. Keyed prerequisites can be specified when adding a task to a workflow using
+`‑[TSKWorkflow addTask:keyedPrerequisiteTasks:]` or 
+`‑[TSKWorkflow addTask:prerequisiteTasks:keyedPrerequisiteTasks:]`. Results from keyed prerequisites
+can be easily retrieved using `‑[TSKTask prerequisiteResultForKey:]`.
+
+Task 1.1 also adds new methods to `TSKTask` so that subclasses can more easily respond to changes in
+a task’s state. See the `SubclassInterface` category of `TSKTask` for more information.
```

Comment:
I’m going to glitz this up a bit and add some code samples. It’s a big deal and should read that way.